staging/src/k8s.io/kubectl/pkg/cmd/portforward/portforward.go

```diff
@@ -141,9 +142,22 @@ func (f *defaultPortForwarder) ForwardPorts(method string, url *url.URL, opts Po
 	if err != nil {
 		return err
 	}
+	fw.ErrorHandler = portForwardErrorHandler
 	return fw.ForwardPorts()
 }
 
+// portForwardErrorHandler handles errors that occur during an accepted port forward connection, so that
+// "connection reset by peer" can be treated as non-fatal. If the error ends in "connection reset by peer", this
+// function logs it and returns true, indicating that the error was handled, and allowing the port forwarder to continue
+// listening for and accepting new connections.
+func portForwardErrorHandler(err error) bool {
+	if err != nil && strings.HasSuffix(strings.ToLower(err.Error()), "connection reset by peer") {
```

Comment:
lowercase suffix-only match seems super rigid... what layer is producing this error and are we sure there's no layer in between kubectl and that layer that will wrap it or add a suffix?